packages/api/db/seeds/examples/users.ts

```diff
@@ -1,12 +1,15 @@
+type Role = "none" | "member" | "admin";
```

Comment:
Can you change the "none" into "unverified"? It would look weird on the fe if you had under the name "None Level foo".